### PR TITLE
Make pp_sql work if rewrite_to_sql_method = false

### PR DESCRIPTION
**Pull request by dputtick**
Second try at this! @kvokka sorry about #18, and sorry this took me so long to get back to. My fault for forgetting to run all of the tests one more time before pushing. I've added a few more tests this time, and they all pass :)

### CODE REVIEW

File: pp_sql.gemspec [hunk 1]
```diff
@@ -25,5 +25,5 @@ Gem::Specification.new do |s|
   s.add_development_dependency 'minitest-focus'
   s.add_development_dependency 'minitest-reporters'
   s.add_development_dependency 'rails', '>= 4.2'
-  s.add_development_dependency 'sqlite3', '~> 1.3.6'
+  s.add_development_dependency 'sqlite3'
```
Comment (dputtick) [inline pp_sql.gemspec:28]: Pinning this gem to 1.3.6 was stopping me from running all of the tests. I think it's a version conflict with the Rails sqlite adapter?

**Comment by kvokka** [comment]
@dputtick Thank you for your PR. Will be happy to merge after making hound happy (or pls describe why it's not possible)

**Comment by dputtick** [comment]
@kvokka ok, I think the things Hound flagged are fixed

**Comment by kvokka** [comment]
Grate! Thank u

**Comment by kvokka** [comment]
@dputtick Had to revert it, because of https://travis-ci.org/github/kvokka/pp_sql/builds/711223824 (and the error message shows that using old version of sqlite was on purpose).

Can you please take a look?

**Comment by dputtick** [comment]
I didn't realize how thorough the Rails/Ruby compatibility testing is! I'll try to see what's going on.

**Pull request merged**